src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/LinkCaseSelectCase.java

```diff
@@ -1,20 +0,0 @@
-package uk.gov.hmcts.sptribs.caseworker.event.page;
-
-import uk.gov.hmcts.sptribs.ciccase.model.CaseData;
-import uk.gov.hmcts.sptribs.ciccase.model.CicCase;
-import uk.gov.hmcts.sptribs.common.ccd.CcdPageConfiguration;
-import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
-
-public class LinkCaseSelectCase implements CcdPageConfiguration {
```

Comment:
Doesn't seem to be related to case flag feature. If not, revert back.

Reply:
Will revisit the changes added as part of merge, will clean-up the case linking related changes as they should not be part of this PR.